android/src/main/java/com/revenuecat/purchases/hybridcommon/paywalls.kt

```diff
@@ -0,0 +1,12 @@
+package com.revenuecat.purchases.hybridcommon
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentActivity
+
+fun presentPaywallFromFragment(fragment: FragmentActivity, requiredEntitlementIdentifier: String?) {
```

Comment:
Should `requiredEntitlementIdentifier` have a default value?

Reply:
I guess the main problem with this approach is that we can't share the result of the fragment (whether the user purchased/cancelled mostly) back to the caller... But I guess we can ask the developers to use `getCustomerInfo` directly if they need to know the result for now.

Reply:
Yeah I was going to work on that next. I was planning on passing an observer type here?

Reply:
That probably won't work, it's not easy to pass information back from a fragment/activity unfortunately: I recommend reading this, it's dense, but it explains some of the options available: https://developer.android.com/guide/fragments/communicate.

The most common ways to share information between fragments are:
- Use the underlying activity, but we don't have a good way to store data there since we are not the owners.
- Use the fragment result APIs, but again, that will depend on the caller to make sure they handle the result correctly.
I'm not sure how this is handled in RN TBH, so maybe they have another recommended way...